spec/active_record/annotate/file_spec.rb

```diff
@@ -78,7 +78,11 @@ class User < ActiveRecord::Base
   end
   
   let(:file) { ActiveRecord::Annotate::File.new(file_path) }
-  let(:configurator) { ActiveRecord::Annotate::Configurator.new }
+  let(:configurator) {
+    c = ActiveRecord::Annotate::Configurator.new
+    c.debug = true
+    c
```

Comment:
This can be simplified to `ActiveRecord::Annotate::Configurator.new.tap { |c| c.debug = true }`.